catalog/task/create-internal-request/0.1/create-internal-request.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-internal-request
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Submit a build request to add operator bundles to an index image
+  params:
+    - name: BINARY_IMAGE
+      type: string
+      description: FBC Image to be added to index image
+    - name: FBC_FRAGMENT
+      type: string
+      description: FBC fragment built by HACBS
+    - name: FROM_INDEX
+      type: string
+      description: Index image (catalog of catalogs) the FBC fragment will be added to    
+    - name: OVERWRITE_FROM_INDEX
+      type: string
+      description: Boolean indicating if the from_index should be overwritten      
+    - name: BUILD_TAGS      
+      type: string
+      description: List of additional tags the internal index image copy should be tagged with
+  steps:
+    - name: create-internal-request
+      image: quay.io/hacbs-release/release-utils:latest
+      script: |
+        #!/usr/bin/env bash
+        # 
+        PATH=/bin:/usr/bin:/usr/local/bin
+        KUBECONFIG=/mnt/kube/config
+
+        export PATH KUBECONFIG
+
+        resourceRequest="/tmp/$$"
+        cat > ${resourceRequest} <<YAML
+        ---
+        apiVersion: appstudio.redhat.com/v1alpha1
+        kind: InternalRequest
+        metadata:
+          name: ir-$(context.pipelineRun.name)
+        spec:
+          params:
+            - name: request
+              value: ir-$(context.pipelineRun.name)
+            - name: params
+              value:
+                - name: binaryImage
+                  value: $(params.BINARY_IMAGE)
+                - name: fbcFragment
+                  value: $(params.FBC_FRAGMENT)
+                - name: fromIndex
+                  value: $(params.FROM_INDEX)
+                - name: overwriteFromIndex
+                  value: $(params.OVERWRITE_FROM_INDEX)
+                - name: buildTags
+                  value: $(params.BUILD_TAGS)
+        YAML
+        oc create -f ${resourceRequest}
```

Comment:
This works? I don't see us installing openshift cli in our Dockerfile

Reply:
Now should work. I added `kubectl` installation to the Dockerfile.